gtsfm/two_view_estimator.py

```diff
@@ -304,6 +324,19 @@ def aggregate_frontend_metrics(
         [report.inlier_ratio_gt_model == 1.0 for report in two_view_reports_dict.values() if report is not None]
     )
 
+    calibrated_configuration_percent = configuration_counts[TwoViewConfigurationType.CALIBRATED] / num_image_pairs * 100
+    planar_or_panoramic_configuration_percent = (
+        configuration_counts[TwoViewConfigurationType.PLANAR_OR_PANORAMIC] / num_image_pairs * 100
+    )
+    degenerate_configuration_percent = configuration_counts[TwoViewConfigurationType.DEGENERATE] / num_image_pairs * 100
+
+    logger.debug(
```

Comment:
This can be info if its a summary.

Reply:
thanks, I updated to `logger.info()` now